src/app/frontend/logs/logs.html

```diff
@@ -71,6 +71,25 @@
         </md-icon>
       </md-button>
       <md-button class="kd-logs-toolbar-button"
+                 ng-click="ctrl.toggleLogFollow()">
+        <md-icon md-font-library="material-icons"
```

Comment:
`md-font-library` is obsolete as we are only using `material-icons` set and it is our default.